starcounter-include.html

```diff
@@ -106,12 +126,18 @@
                     const starcounterInclude = this;
                     this.defaultComposition = null;
                     const importedTemplate = document.createElement('imported-template');
-                    importedTemplate.addEventListener('stamping', function fetchCompositions(event) {
+                    importedTemplate.addEventListener('stamping', async function fetchCompositions(event) {
                         var mergedComposition = null;
                         const templates = event.detail.querySelectorAll('template[is="declarative-shadow-dom"]');
                         if(templates.length){
                             mergedComposition = document.createDocumentFragment();
                             for(const individualComposition of templates) {
+                                const styleSheetLinks = individualComposition.content.querySelectorAll('link[rel="stylesheet"]');
+                                for(const link of styleSheetLinks) {
+                                    const style = await fetchStyle(link.href);
```

Comment:
Isn't it making a cascade of fetches? One requests wait for another instead of bulking them.